_scripts/main.js

```diff
@@ -19,21 +19,19 @@ function hamburgerColor() {
 
   $hamburgerIcon.addClass('icon-white');
 
-  $(document).scroll(function () {
+  $(document).scroll(function() {
     var scrollPos = window.scrollY;
 
     for (var i = 0; i < $projectSectionArr.length; i++) {
       var $projectSection = $projectSectionArr.eq(i);
-      var offset = $projectSection.offset();
-      var sectionHeight = $projectSection.innerWidth();
-      if (scrollPos > offset && scrollPos < sectionHeight) {
-      //   console.log('hello');
-      //   console.log(offset);
-      //   if ($projectSection.hasClass(.dark)) {
-      //     $hamburgerIcon.addClass('icon-white');
-      //   } else {
-      //     $hamburgerIcon.removeClass('icon-white');
-      //   }
+      var offset = $projectSection.offset().top;
+      var sectionHeight = $projectSection.innerHeight();
+      if (scrollPos > offset && scrollPos < offset + sectionHeight) {
```

Comment:
we don't just wanna check if the scroll position is less than the section height, that means that the scroll position is above the section, we want offset + sectionHeight, now we're testing if it's between `offset` and `offset + sectionHeight`